src/main/java/com/checkmarx/flow/CxFlowRunner.java

```diff
@@ -175,6 +183,16 @@ private void commandLineRunner(ApplicationArguments args) throws ExitThrowable {
             exit(1);
         }
 
+        if(breakBuild != null && !breakBuild.isEmpty()) {
+            if (StringUtils.containsIgnoreCase("true", breakBuild)){
+                breakBuildArgument = true;
+            }
+            else{
+                breakBuildArgument = false;
+            }
```

Comment:
don't write this type of if. do it like this:
breakBuildArgument = StringUtils.containsIgnoreCase("true", breakBuild)